dev/refresh_source.py

```diff
@@ -523,6 +539,10 @@ def init_argument_parser(cls, parser):
                                ' otherwise skip it.'
                                ' If cloning, then clone this branch.')
 
+      parser.add_argument('--allow_master', default=False, action='store_true',
```

Comment:
maybe `fallback_master` or `default_master`? `allow` seems like it would deal with permissions.